core/src/main/java/cz/o2/proxima/repository/ConfigRepository.java

```diff
@@ -1129,6 +1210,24 @@ private void loadSingleFamily(String name, boolean overwrite, Map<String, Object
     insertFamily(family.build(), overwrite);
   }
 
+  private StorageType getStorageType(Map<String, Object> cfg, boolean isTransactional) {
+    StorageType type =
+        Optional.ofNullable((String) cfg.get(TYPE))
+            .map(StorageType::of)
+            .orElse(isTransactional ? StorageType.PRIMARY : null);
+    Preconditions.checkArgument(type != null, "Missing required setting %s", TYPE);
+    return type;
+  }
+
+  private AccessType getAccessType(Map<String, Object> cfg, boolean isTransactional) {
+    return AccessType.from(
+        Optional.ofNullable(cfg.get(ACCESS))
+            .orElse(
+                isTransactional
+                    ? Lists.newArrayList(COMMIT_LOG, STATE_COMMIT_LOG, CACHED_VIEW)
```

Comment:
I'm not sure about this hidden magic (same as for `PRIMARY`) where just enable transaction will leads to some default values (where for example STATE_COMMIT_LOG can have consequences to kafka topic settings). Maybe we can leave it to user and just validate settings (with reasonable exception).

Reply:
The `isTransactional` refers to the case, where entity is `_transaction`, so this does not affect any other families.

Reply:
Ok, I see.  `isTransactional` was a little bit confusing for me